# this is a timestamped message that contains an array of motor control information

Header header
Info info
Motor[] motors
Sensor[] sensors
Imu[] imus
Node[] nodes
Trajectory trajectory
string[] actions